### PR TITLE
Cross-mio updates for Pioneer 16

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes cross-mio updates to the following:
eng_bulk_load-AssetRecord.csv
Addition of UIDs for new platcon on CP03ISSM-13:
CGCON-BCPM01-50015, CGCON-BDCL11-50015, CGCON-BDCL12-50015
Addition of new STC endcap on CP02PMUO-17:
CGCON-BSTC01-00043

sensor_bulk_load-AssetRecord.csv
Addition of new MOPAK on CP02PMUO-17:
CGINS-MOPAK0-70027

CruiseInformation.csv: Addition of Pioneer 16 cruise - AR52

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 16 AM Updates [DT-127]](https://trello.com/c/8zHulE3Q/127-pioneer-16-am-updates-dt-127)

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -107,6 +110,7 @@ CGCON-BSTC01-00017,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and
 CGCON-BSTC01-00018,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0018,,,,,Woods Hole Oceanographic Institution
 CGCON-BSTC01-00040,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0040,,,,,Woods Hole Oceanographic Institution
 CGCON-BSTC01-00042,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0042,,,,,Woods Hole Oceanographic Institution
+CGCON-BSTC01-00043,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,3701-00143-00000-00043,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/eng_bulk_load-AssetRecord.csv:113]: The S/N should be 3701-00143-00001-00043  (it's missing the 1 in the third set of numbers).

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -1858,6 +1858,7 @@ CGINS-MOPAK0-44016,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6
 CGINS-MOPAK0-58478,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-58478,,,,,
 CGINS-MOPAK0-58481,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-58481,,,,,
 CGINS-MOPAK0-58484,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-58484,,,,,
+CGINS-MOPAK0-70027,,Sensor,0,3-Axis Motion Pack: MOPAK,,MicroStrain,3DM-GX3-25,6223-4220-70027,,,,,
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1861]: In RDB the vendor S/N is listed as 6253-4220.70027.  Should it be a decimal or a dash.  This is a bigger issue we can address later.

File: cruise/CruiseInformation.csv [hunk 3]
```diff
@@ -240,3 +240,4 @@ EK20210120,Elakha,2021-01-20T16:40:00,2021-01-20T23:50:00,Endurance glider deplo
 TS20210131,Tenacious II,2021-01-31T08:00:00,2021-02-01T03:00:00,Pioneer 15b glider deployments
 WJ20210209,Warren Jr.,2021-02-09T22:00:00,2021-02-11T13:00:00,Pioneer glider cp_340 recovery
 EK20210210,Elakha,2021-02-10T17:15:00,2021-02-11T01:00:00,Endurance glider recoveries
+AR52,Neil Armstrong,2021-03-29T00:00:00,2021-04-18T23:59:00,Pioneer 16
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:243]: Looks good.

**Review by snwhite — CHANGES_REQUESTED** [review]
STC full S/N needs to be corrected.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -107,6 +110,7 @@ CGCON-BSTC01-00017,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and
 CGCON-BSTC01-00018,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0018,,,,,Woods Hole Oceanographic Institution
 CGCON-BSTC01-00040,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0040,,,,,Woods Hole Oceanographic Institution
 CGCON-BSTC01-00042,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,SWE 0042,,,,,Woods Hole Oceanographic Institution
+CGCON-BSTC01-00043,,Sensor,0,Sensor and Telemetry Platform Controller,Sensor and Telemetry Controller,WHOI,STC,3701-00143-00001-00043,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/eng_bulk_load-AssetRecord.csv:113]: Correction looks good!

**Review by snwhite — APPROVED** [review]

**Review by cdobs — APPROVED** [review]
Everything looks good to me

**Pull request merged**